geoAssembler/geometry.py

```diff
@@ -194,15 +194,25 @@ def write_quad_pos(self, filename):
     def wrap_extra_geom(geom_obj):
         """Wrap an EXtra-geom geometry object in the appropriate class
         """
+        filename = None
         if isinstance(geom_obj, GeometryAssembler):
             return geom_obj
+
         elif isinstance(geom_obj, AGIPD_1MGeometry):
             return AGIPDGeometry(geom_obj)
```

Comment:
Not a code improvement suggestion, but a more general comment: I'm aware the different treatment if AGIPD is mainly due to the fact that we (DA, SPB) typically don't use HDF5 files, but rather CrystFEL geom files - but could we think of cases were HDF5 is still useful? It seems MID are using HDF5 geom for PyFAI notebooks ...

Reply:
Yeah, I'm trying to get away from the assumption that we always use `.geom` files for AGIPD and always HDF5 files for DSSC and LPD. I've done some work on that as part of #30, though it doesn't cover AGIPD <-> HDF5 yet.

Reply:
While I'll be looking at #30 as well, the comment is more to have a written note for the backlog, thanks for the answer (should we/I add that to the issues?)

Reply:
Yup, particularly if you're aware of a specific need (e.g. adjusting AGIPD quadrant positions together with an EuXFEL HDF5 geometry file), please do make an issue.

If you know which interface the users in question would prefer to use, please make a note of that, too. :slightly_smiling_face:

Reply:
Yes, I can ask MID even today, I guess, at the weekly ICI meeting (I strongly assume Johannes & Co, resp. their users have an affinity for Jupyter). Accordingly I can formulate an issue.

Reply:
Thanks!